app/views/overview/_service-instance-row.html

```diff
@@ -6,8 +6,8 @@
     <div class="list-pf-content">
       <div class="list-pf-name">
         <h3>
-          <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
-          <div ng-bind-html="row.apiObject.metadata.name | highlightKeywords : row.state.filterKeywords" class="list-row-longname"></div>
```

Comment:
Yeah, it looks like there was a change to a dependency. The diff should disappear if you rebase after #1471 merges